opengamedata/opengamedata-website
# FD-32 Date default

- set default start date to start date from file_list instead of game usage api
- Remove unnecessary call to `getGameUsage` in gamedata.php - the js file calls it onload. 
- Update when month & usage labels update in game_usage.js for a more responsive user experience. 
- Hide Publications section if no publications exist

Testing: 
- Add new game AQUALAB to data/game_list.json - can either change the key for an existing game to "AQUALAB" or duplicate an existing game with the new key. This will give you a game with usage data & file_list data for testing.
- Verify game data page loads and displays as expected, including month nav and all month names.
- Verify Pipeline files download the correct month's data when available.
- Test page with games with no data, e.g., WAKE and verify all areas fail gracefullly.
- Test page with game with file data only, e.g., LAKELAND, which should default to the most recent data with file data - 8/2021.
